doc/interop-test-descriptions.md

```diff
@@ -39,6 +39,9 @@ Clients should accept these arguments:
 * --service_config_json=SERVICE_CONFIG_JSON
     * Disables service config lookups and sets the provided string as the
       default service config.
+* --additional_metadata=ADDITIONAL_METADATA
+    * Additional metadata to send in each request, as a semicolon-separated list
```

Comment:
Have some mention that the keys must be ASCII-only. No `-bin` headers allowed?

Also make clear the first colon separates the key and value.

Reply:
Done